File: 오선민/programmers/P181866.py
'''
문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181866
시간: 5.31ms
메모리: 12.3MB
PR 제목 : 날짜 사이트_명 문제_번호 사용_언어
'''

def solution(myString):
    # sample_list = list(filter(None, sample_list))
    return sorted(list(filter(None, myString.split("x"))))

File: 공예진/programmers/P181866.java
//문제: https://school.programmers.co.kr/learn/courses/30/lessons/181866
//시간: 20.06ms
//메모리: 103MB

import java.util.*;

class Solution {
    public String[] solution(String myString) {
        String[] arr = myString.split("x");
        Arrays.sort(arr);

        List<String> list = new ArrayList<>();
        for (int i = 0; i < arr.length; i++) {
            if (!arr[i].equals("")) {
                list.add(arr[i]);
            }
        }

        String[] answer = new String[list.size()];
        for (int i = 0; i < list.size(); i++) {
            answer[i] = list.get(i);
        }

        return answer;
    }
}